base/docs/helpdb/Base.jl

```diff
@@ -130,13 +130,28 @@ Obtain the contents of an `IOBuffer` as an array, without copying. Afterwards, t
 takebuf_array
 
 """
-    download(url,[localfile])
+    download(url [localfile] [silent=false])
 
 Download a file from the given url, optionally renaming it to the given local file name.
+Passing silent=true will suppress progress indicators.
 Note that this function relies on the availability of external tools such as `curl`, `wget`
 or `fetch` to download the file and is provided for convenience. For production use or
 situations in which more options are needed, please use a package that provides the desired
-functionality instead.
+functionality instead. Platform specific details: In windows, download requires a full URL,
```

Comment:
Please move `Platform specific details:` to a separate paragraph, as a title like this: `# Platform-Specific Details`. I would also rephrase it as something like "On Windows, a full URL must be provided, including the protocol prefix (e.g. "http://" for HTTP)."